packages/main-ui/src/pages/politician/proclaim_popup.rs

```diff
@@ -0,0 +1,114 @@
+#![allow(non_snake_case)]
+use dioxus::prelude::*;
+use dioxus_translate::*;
+use dioxus_popup::PopupService;
+use crate::{
+    theme::Theme, 
+    components::{
+        checkbox::Checkbox,
+        dropdown::Dropdown,
+    },
+};
+use super::i18n::PoliticianStanceTranslate;
+
+#[component]
+pub fn ProclaimPopup(
+    #[props(default = "proclaim_popup".to_string())] id: String,
+    #[props(default = "".to_string())] class: String,
+    name: String,
+    party: String,
+    stance: String,
+    lang: Language,
+) -> Element {
+    let theme_service: Theme = use_context();
+    let theme = theme_service.get_data();
+    let tr = translate::<PoliticianStanceTranslate>(&lang);
+    let mut popup: PopupService = use_context();
+    let mut agreed = use_signal(|| false);
+    let mut stance_signal = use_signal(|| stance);
+
+    rsx! {
+        div { id, class,
+            div { class: "flex flex-col w-full items-start justify-start gap-[35px] pt-[10px]",
+                div { class: "flex flex-col w-full gap-[10px]",
+
+                    // NAME
+                    div { class: "flex flex-col w-full gap-[2px]",
```

Comment:
```suggestion
                    div { id: "proclaim-popup-name", class: "flex flex-col w-full gap-[2px]",
```
이런식으로 코멘트 대신에 id 로 만들어주세요. 그러면 나중에 테스트 코드에서도 직관적으로 테스트가 가능해집니다.